sw/device/lib/testing/mock_mmio.h

```diff
@@ -141,6 +141,22 @@ Int ToInt(std::initializer_list<BitField> fields) {
  */
 inline internal::LittleEndianBytes LeInt(const char *bytes) { return {bytes}; }
 
+/**
+ * Returns an integer with the bit at offset set to 1 and all other bits to 0.
+ */
+template <typename Int>
+Int ToIntBit(Int offset) {
+  return (1 << offset);
+}
+
+/**
+ * Returns an integer with the bit at offset set to 0 and all other bits to 1.
+ */
+template <typename Int>
+Int ToIntBitInverted(Int offset) {
+  return ~(1 << offset);
+}
+
```

Comment:
How are these different from the existing `bitfield_bit32_write`? I can express these as

```cc
bitfield_bit32_write(0, index, true);

bitfield_bit32_write(-1, index, false);
```

It's not immediately clear what benefit this provides now. If the intent is to do something like googletest matcher (alluded to in my other comment), these functions should probably be called `Bit` and `InvertedBit`, though I would almost feel like writing `EXPECT_READ(offset, ~Bit(index))` is more clear to the reader?